client/Components/header.jsx

```diff
@@ -5,36 +5,76 @@ class Header extends React.Component {
+    const lastWord = words.pop();
+    const nameSansLastWord = words.join(' ');
+    return (
+      <h1>
+        {nameSansLastWord + ' '}
+        <span className='last-word-check'>{lastWord}<span className='verified-site fa fa-check' /></span>
+      </h1>
+    )
+  }
+
+  showNearby() {
+    return (
+      <div className='nearby'>
+
+      </div>
+    )
+  }
+
   render() {
-    if (!this.props.header.errorOccured) {
-      return (
-        <div className='header'>
-          <div className="recommend-percentage">
-            <span className="icon fa fa-thumbs-up"></span> {Math.trunc(100 * this.props.header.percentRec)}% <span className="recommend-text">Recommend</span>
-          </div>
-          <div className='header-buttons'>
-            <div className='upload'>
-              <a className='button upload-photo'><span className='icon fa fa-camera'></span> Upload</a>
-            </div>
-            <a className='button save-listing'>Save to list<i className="fas fa-chevron-down chevron"></i></a>
-            <a className='button share'><span className='icon hc-awesome-share-apple'/></a>
-          </div>
+    // console.log(this.props)
+    return (
+      <div className='two-thirds'>
+        <div className="breadcrumb">
+          <li>
+            <a className="underlined" href="https://www.hipcamp.com/en-US/discover/united-states">{this.props.location.country}</a>
+          </li>
+          <li>
+            <a className="underlined" href={`https://www.hipcamp.com/en-US/discover/${this.props.location.state}`}>{this.props.location.state}</a>
+          </li>
+          <li>
+            <a className="underlined" href={`https://www.hipcamp.com/en-US/discover/${this.props.location.state}`} style={{ display: 'none' }}>{this.props.siteName}</a>
+          </li>
         </div>
-      )
-    } else {
-      console.log('Not connected to reviews service');
-      return (
+
+        {this.props.location.verified ? this.verifiedSite() : <h1>{this.props.siteName}</h1>}
+
+
+          {this.props.header.nearby
+            ? this.showNearby()
+            : <div className='nearby'><span className='nearby-strong'>Nearby: </span>... Coming Soon!</div>
+          }
+
+
         <div className='header'>
+          {// checking error in accessing review service
+            !this.props.header.errorOccured
+              ? this.showRecommendation()
+              : <div className='empty' />
+          }
+
           <div className='header-buttons'>
             <div className='upload'>
               <a className='button upload-photo'><span className='icon fa fa-camera'></span> Upload</a>
             </div>
             <a className='button save-listing'>Save to list<i className="fas fa-chevron-down chevron"></i></a>
-            <a className='button share'><span className='icon hc-awesome-share-apple'/></a>
+            <a className='button share'><span className='icon hc-awesome-share-apple' /></a>
```

Comment:
Your UI is looking really good. My main Q here is do you need to enable any user interaction with your service? Are any of your buttons clickable. What should happen if I click 'expand' at the bottom of the 'essentials' etc. components?